imagetest/test_suites/network/mtu_test.go

```diff
@@ -0,0 +1,67 @@
+package network
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	gceMTU                 = 1460
+	defaultInterface       = "eth0"
+	defaultDebianInterface = "ens4"
+)
+
+var (
+	runtest = flag.Bool("runtest", false, "really run the test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if *runtest {
+		os.Exit(m.Run())
+	} else {
+		os.Exit(0)
+	}
+}
+
+func TestDefaultMTU(t *testing.T) {
+	var networkInterface string
+
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("couldn't get image from metadata")
+	}
+
+	switch {
+	case strings.Contains(image, "debian-10"):
+		networkInterface = defaultDebianInterface
```

Comment:
if this is really only on deb10, maybe name the constant debian10Interface ?
how many images did you test this on, btw? our other code finds the default interface by checking what metadata thinks is default, you might consider doing here